config/settings/test.yml

```diff
@@ -5,6 +5,7 @@ govuk_notify:
   course_publish_email_template_id: please_change_me
   course_withdraw_email_template_id: please_change_me
   course_vacancies_filled_email_template_id: please_change_me
+  course_subjects_updated_email_template_id: please_change_me
```

Comment:
@aliuk2012 recently demonstrated that there's no need to keep these template IDs secret. Can we start to just put them into the config here? And we'll have a larger ticket later to change the remaining ones?

Reply:
They should be in the global `settings.yml` file so they apply across all environments

Reply:
Do we need these in the test env still?